src/streaming/MediaPlayer.js

```diff
@@ -2725,6 +2725,9 @@ function MediaPlayer() {
             return null;
         }
 
+        if (value.id) {
```

Comment:
I suggest to rather check for `undefined` here. Otherwise, if `id` is `0` it will be ignored